_inc/client/at-a-glance/backups.jsx

```diff
@@ -126,13 +129,19 @@ class DashBackups extends Component {
 			return renderCard( {
 				className: 'jp-dash-item__is-inactive',
 				status: 'no-pro-uninstalled-or-inactive',
-				content: __(
-					'To automatically back up your entire site, please {{a}}upgrade your account{{/a}}.',
-					{
-						components: {
-							a: <UpgradeLink source="aag-backups" target="at-a-glance" feature="backups" />,
-						},
-					}
+				overrideContent: (
+					<JetpackBanner
+						callToAction={ __( 'Upgrade' ) }
+						title={ __(
+							'Never worry about losing your site – automatic backups keep your content safe.'
+						) }
+						disableHref="false"
+						href={ this.props.upgradeUrl }
+						eventFeature="backups"
+						path="dashboard"
+						plan={ PLAN_JETPACK_PREMIUM }
```

Comment:
Backups are available on personal plans as well though. Should we lower that requirement to personal plans?

Reply:
It needs to be premium to display green; if it is switched to personal then it displays like this: 
<img width="827" alt="image" src="https://user-images.githubusercontent.com/42627630/60044222-82293200-9687-11e9-9d08-0bd16ac28803.png">

So I think we should leave it as premium for now, and next release do some work on the JetpackBanner to change how the colors are selected if that is necessary.

Reply:
Yep. We wanted them to look less like warnings since the plan colors are likely something only we notice.